src/userscores/qml/MuseScore/UserScores/ExportDialog.qml

```diff
@@ -0,0 +1,237 @@
+import QtQuick 2.9
```

Comment:
Please replace to the actual version. Now we use qt 5.15. It means you must use 2.15